com.unity.visualeffectgraph/Editor/FilterPopup/VFXBlockProvider.cs

```diff
@@ -44,37 +45,42 @@ public void CreateComponentTree(List<VFXFilterWindow.Element> tree)
             tree.Add(new VFXFilterWindow.GroupElement(0, title));
             var descriptors = GetDescriptors();
 
+            var depth = 1;
             string prevCategory = "";
-            int depth = 1;
+            string[] prevSplit = new string[0];
+
+            List<T> noCategory = new List<T>();
```

Comment:
You have to be consistent with convention of this current file, so, if it uses "explicit" declaration, it's fine but it should also be `var` [here](https://ono.unity3d.com/unity-extra/unity-meta/files/@/ReferenceSource/CSharp/Assets/CSharpReference.cs#L222)